CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+## 2.15.4
+
+### Fixed
+* Fixed an issue where the `log_level` of `Logger` was changed unintentionally [#222]
+* Fixed an issue cause by a `Logger` not being correctly thrown away, causing an exception. [#220]
+* Added `title` field to assets to prevent Error raising when the field is empty. [#218]
+* Fixed an issue with `ArrayLike` not taking multiple arguments leading to Errors when using it with pagination libraries. [#215]
```

Comment:
Missing the links to the PRs/Issues mentioned, they do not work as links this way.

Reply:
Ah, i just copied from the draft release, let me correct it.

Reply:
corrected.